setup.py

```diff
@@ -53,7 +53,7 @@ def absolute_links(txt):
       url='https://github.com/PEtab-dev/PEtab',
       packages=find_packages(exclude=['doc*', 'test*']),
       install_requires=['numpy>=1.15.1',
-                        'pandas>=1.0.1',
+                        'pandas>=1.2.0',
```

Comment:
does get_handle not exist or behave differently for pandas < 1.2.0? but fine to enforce <=1.2.0 I think.

Reply:
Does exist but may behave differently (the function that PEtab was using was "merged" with `get_handle` in the pandas PR that prompted this PR -- linked above). I haven't tested this PR with previous pandas versions.

Reply:
thanks for the info, then enforcing :panda_face: >= 1.2.0 simply everywhere.